### PR TITLE
Case insensitive filter in RegistryIndex

### PR DESCRIPTION
**Pull request by fakoua**
Case insensitive filter in RegistryIndex

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,6 +35,7 @@
     "query-string": "^6.8.3",
     "react": "^16.10.2",
     "react-apexcharts": "^1.3.3",
+    "react-copy-to-clipboard": "^5.0.2",
```
Comment (ry) [inline package.json]: undo

Comment (fakoua) [reply package.json]: Sorry @ry, what's the issue?

File: src/component/CodeBlock.tsx [hunk 2]
```diff
@@ -11,6 +11,11 @@ import json from "react-syntax-highlighter/dist/cjs/languages/hljs/json";
 import markdown from "react-syntax-highlighter/dist/cjs/languages/hljs/markdown";
 import shell from "react-syntax-highlighter/dist/cjs/languages/hljs/shell";
 import { useDarkMode } from "../hook/theme";
+import FileCopyIcon from '@material-ui/icons/FileCopy';
+import CheckIcon from '@material-ui/icons/Check';
+import { IconButton, Container, Zoom } from "@material-ui/core";
+import { makeStyles } from '@material-ui/core/styles';
+import { CopyToClipboard } from 'react-copy-to-clipboard';
```
Comment (ry) [inline src/component/CodeBlock.tsx]: unrelated changes. please merge master branch.

Comment (fakoua) [reply src/component/CodeBlock.tsx]: @ry 
The branch was merged, the only commit is "Case insensitive filter in RegistryIndex" this shows all the history from the first fork.

Thansk

File: src/page/Registry.js [hunk 3]
```diff
@@ -107,25 +106,23 @@ export default function Registry() {
...
+          </span>
         ) : null}
         {state.rawUrl ? (
-          <p>
-            <Link to={state.rawUrl}>Raw</Link>
-          </p>
+          <span>
+            &nbsp;|&nbsp;<Link to={state.rawUrl}>Raw</Link>
+          </span>
         ) : null}
```
Comment (ry) [inline src/page/Registry.js]: Can you do this in a separate PR?

**Comment by fakoua** [comment]
will create new PR
